website/docs/uninstall/windows-uninstall.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 118
+title: Windows
+description: How to uninstall Podman Desktop and Podman on Windows.
+tags: [podman-desktop, uninstall, Windows]
+keywords: [podman desktop, containers, podman, uninstall, Windows]
+---
+
+# Uninstall Podman Desktop on Windows
+
+Uninstalling Podman Desktop does not automatically remove the created Kubernetes clusters, Podman, or other resources. To uninstall Podman Desktop, perform the following tasks:
+
+1. Optional: Remove cluster resources
+1. Uninstall Podman
+1. Uninstall Podman Desktop
+
+## Optional: Remove cluster resources
+
+- For Kind, use the UI to [delete the Kind cluster](/docs/kind/deleting-your-kind-cluster) or run the `kind delete clusters --all` command.
+- For Minikube, run `minikube delete` and then remove the `~/.minikube` folder.
+
+  :::note
+
+  You can find the installed binaries and plugins for Podman Desktop at `~/.local/share/containers/podman-desktop/extensions-storage/`, and you may delete them if necessary.
+
+  :::
+
+## Uninstall Podman
+
+You can delete all pods, containers, and images by removing the Podman machine.
```

Comment:
We may have, for some rare user multiple machines. To remove **all** machine, we may use `podman machine reset`

```
$: podman machine reset -h
Remove all machines

Description:
  Remove all machines, configurations, data, and cached image
```